### PR TITLE
C359585 Verify clicking on the "Commit changes" button

### PR DESCRIPTION
**Pull request by MaksimTerekhov**
https://foliotest.testrail.io/index.php?/cases/view/359585

### CODE REVIEW

**Review by OllIsanbaev — APPROVED** [review]

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-user-uuids.cy.js [hunk 1]
```diff
@@ -137,5 +137,24 @@ describe('bulk-edit', () => {
       BulkEditActions.openInAppStartBulkEditFrom();
       BulkEditActions.verifyCalendarItem();
     });
+
+    it('C359585 Verify clicking on the "Commit changes" button (firebird)', { tags: [testTypes.smoke, devTeams.firebird] }, () => {
```
Comment (penfriend) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-user-uuids.cy.js:141]: It looks like that there is no need in permission: "Users: Can view user profile". So we can remove this permission from the test case

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-user-uuids.cy.js [hunk 2]
```diff
@@ -137,5 +137,24 @@ describe('bulk-edit', () => {
...
+      BulkEditActions.verifyCheckedDropdownMenuItem();
+      BulkEditActions.verifyUncheckedDropdownMenuItem();
+
+      BulkEditActions.openInAppStartBulkEditFrom();
+      BulkEditActions.fillPatronGroup('graduate (Graduate Student)');
+      BulkEditActions.confirmChanges();
+      BulkEditActions.verifyAreYouSureForm(1, user.username);
+      BulkEditActions.commitChanges();
+      BulkEditSearchPane.waitFileUploading();
+      BulkEditActions.verifySuccessBanner(1);
```
Comment (penfriend) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-user-uuids.cy.js:157]: It is better to check if the value is been updated also. There is a note about it in the expected result column on the step 8 in the test case: The Preview of record changed is shown with only updated values

**Review by penfriend — APPROVED** [review]

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-user-uuids.cy.js (hunk 1 above)
Comment (MaksimTerekhov) [reply cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-user-uuids.cy.js:141]: done

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-user-uuids.cy.js (hunk 2 above)
Comment (MaksimTerekhov) [reply cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-user-uuids.cy.js:157]: done

**Pull request merged**